components/membership/registration-impl/src/main/kotlin/net/corda/membership/impl/registration/dynamic/member/DynamicMemberRegistrationService.kt

```diff
@@ -283,12 +287,19 @@ class DynamicMemberRegistrationService @Activate constructor(
                 val roles = MemberRole.extractRolesFromContext(context)
                 val notaryKeys = generateNotaryKeys(context, member.shortHash.value)
                 logger.debug("Member roles: {}, notary keys: {}", roles, notaryKeys)
+                val groupReader = membershipGroupReaderProvider.getGroupReader(member)
+                val previousContext = groupReader
+                    .lookup(member.x500Name)
```

Comment:
This should be:
```suggestion
                    .lookup(member.x500Name, MembershipStatusFilter.ACTIVE_OR_SUSPENDED)
```
if you want to have the active or suspended information for given member.